erp/schema.py

```diff
@@ -642,17 +645,30 @@
             PERSONNELS_AUCUN,
         ],
     },
+    "accueil_equipements_malentendants_presence": {
+      "is_a11y": True,
+      "label": "Équipement(s) de communication présence",
+      "help_text": mark_safe(
+        "L'accueil est-il équipé de produits ou prestations dédiés aux personnes sourdes ou malentendantes (boucle à induction magnétique, langue des signes française, solution de traduction à distance, etc)"
+      ),
+      "help_text_ui": mark_safe(
+        "Présence d'équipements ou de prestations facilitant la communication avec les personnes sourdes ou malentendantes"
+      ),
+      "section": SECTION_ACCUEIL,
+      "nullable_bool": True,
+      "warn_if": False,
+    },
     "accueil_equipements_malentendants": {
         "is_a11y": True,
         "label": "Équipement(s) de communication",
         "help_text": mark_safe(
-            "L'accueil est-il équipé de produits ou prestations dédiés aux personnes sourdes ou malentendantes (boucle à induction magnétique, langue des signes française, solution de traduction à distance, etc)"
+            "Équipements disponibles"
```

Comment:
I'd go with `Équipements sourds/malentendants disponibles`. Thoughts?

Reply:
Agree, was really not sure what to put here